scripts/pkglib.sh

```diff
@@ -164,6 +164,33 @@ function checkout_repo()
     cp ../scripts/apport_hook.py $OBS_WORKDIR/$APPORT_HOOK
 }
 
+function generate_kernel_config() {
+    local kernel_version=$1
+    local kernel_filename=$2
```

Comment:
Could you add checks to these params (check strings aren't empty, file exists, etc)?